File: frontend/src/app/models/ContenitoreDatiRegistrazioneAzienda.ts
import {AdminAziendaleDTO} from '@models/adminAziendaleDTO.models';
import {AziendaDTO} from '@models/aziendaDTO';

export interface ContenitoreDatiRegistrazioneAzienda {
  adminAziendale: AdminAziendaleDTO;
  azienda: AziendaDTO;
}

File: backend/src/main/java/it/unical/fleetgo/backend/Models/DTO/ContenitoreDatiRegistrazioneAzienda.java
package it.unical.fleetgo.backend.Models.DTO;

import it.unical.fleetgo.backend.Models.DTO.Utente.AdminAziendaleDTO;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ContenitoreDatiRegistrazioneAzienda {
    public AziendaDTO azienda;
    public AdminAziendaleDTO adminAziendale;
}